src/score.js

```diff
@@ -0,0 +1,61 @@
+const fs = require("fs");
+const scoreFile = "./score.json";
+
+async function processScore(botContext) {
+  const score = botContext.sessions[botContext.chatID].scoreInSession;
+  const scoreBoard = await readScoreFromFile();
+
+  if (botContext.chatID in scoreBoard) {
+    if (score > scoreBoard[botContext.chatID].hiScore) {
+      scoreBoard[botContext.chatID].hiScore = score;
+      botContext.hiScore = score;
+    }
+  } else {
+    scoreBoard[botContext.chatID] = { hiScore: score, userName: botContext.userName };
+    botContext.hiScore = score;
+  }
+  await saveScoreToFile(scoreBoard);
+}
+
+async function getHiScore(botContext) {
```

Comment:
getHiScore is probably not the best name. You might mean `getHighScore`?

Reply:
fixed